src/coreclr/src/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -484,8 +537,14 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                                                     INS_OPTS_NONE);
                     }
                 }
-            }
-            break;
+                break;
+
+            case NI_AdvSimd_AddWideningLower:
+            case NI_AdvSimd_AddWideningUpper:
+            case NI_AdvSimd_SubtractWideningLower:
+            case NI_AdvSimd_SubtractWideningUpper:
+                GetEmitter()->emitIns_R_R_R(ins, emitSize, targetReg, op1Reg, op2Reg, opt);
```

Comment:
I believe this is identical to `case NI_Crc32_ComputeCrc32:` et al. Is there a reason you didn't combine it? Is the idea to keep them in order, and rely on the C++ compiler to de-duplicate the code? (I see there's some duplication already).

Reply:
No, I just missed this. Thanks for spotting this. 

I will follow up and de-duplicate this in a separate PR and I also need to address one of your another suggestions [here](https://github.com/dotnet/runtime/pull/35030#discussion_r412473191)